contracts/forwarder/Forwarder.sol

```diff
@@ -8,7 +8,7 @@ import "./IForwarder.sol";
 contract Forwarder is IForwarder {
     using ECDSA for bytes32;
 
-    string public constant GENERIC_PARAMS = "address from,address to,uint256 value,uint256 gas,uint256 nonce,bytes data";
+    string public constant GENERIC_PARAMS = "address from,address to,uint256 value,uint256 gas,uint256 nonce,bytes data,uint256 validUntil";
```

Comment:
I believe placement of 'bytes' parameter in the end of the struct was deliberate. Is there a downside to adding 'validUntil' field in the middle, with all the uint256 parameters?